csi/csi.go

```diff
@@ -0,0 +1,76 @@
+/*
+CSI Interface for OSD
+Copyright 2017 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package csi
+
+import (
+	"net"
+	"sync"
+
+	//"github.com/libopenstorage/openstorage/api/spec"
+	csi "github.com/container-storage-interface/spec/lib/go/csi"
```

Comment:
The alias should not be required as the package itself is named csi

Reply:
Yeah, this is carry over from the generated code. I'll take care of it.